doc/userguide/output/eve/eve-json-output.rst

```diff
@@ -266,6 +266,25 @@ enabled, then the log gets more verbose.
 
 By using ``custom`` it is possible to select which TLS fields to log.
 
+MQTT
+~~~~
+
+EVE-JSON output for MQTT consists of one object per MQTT transaction, with some common and various type-specific fields.
+Two aspects can be configured:
+
+YAML::
+
+        - mqtt:
+            # passwords: yes           # enable output of passwords
+            # msg-log-limit: 0         # limit size of logged messages in bytes.
+                                       # Can be specified in kb, mb, gb. Just a number
+                                       # is parsed as bytes. Default is 0 (no limit)
```

Comment:
I think we should have another default than no limit.

Should the default be rather to not log the message at all ?

Reply:
But it's the current behavior, right? In that case, if we agree to change it, shouldn't we add a note to upgrading?

Reply:
Yes, it is the current behavior...

Reply:
I've also always seen this as an optional thing rather than a mandatory limit.
Note that there is already a separate limit with a set default for the whole of the message, which already limits the impact of huge messages not only for logging, but also for detection.

Reply:
But I'd totally be fine with setting a default here. Let's pick one. 1KB? 8KB?

Reply:
Picked 1KB as new default and also as the one to use in alerts.